Codigo/Problemas/steinerTree

```diff
@@ -0,0 +1,66 @@
+// Steiner Tree
+//
+// encontra a arvore de menor peso que cobre os vertices S e retorna o peso e a solução
+// descomenta os vw's e coloca dentro dos parenteses para ter peso nos vertices
+//
+// k = |S|
+//  O(3^k * n + 2^k * m log m)
+
+vector<pair<int, long long>> g[MAX]; // {vizinho, peso}
+long long d[1 << K][MAX]; //[2^k][n]
+//int vw[MAX];
+int n;
+
+pair<ll,vector<pair<int,int>>> steiner(const vector<int> &S){
+    int k = S.size();k--;
+    for(int mask = 0; mask < (1 << k); mask++)for(int v = 0; v < n; v++) d[mask][v] = LINF;
```

Comment:
Você pode por favor formatar igual o resto da lib? Em particular

for (int i = 0; i < n'i++) {

(espaço entre o for e o abre parentesis)